docs/experiments-vector-stores.md

```diff
@@ -0,0 +1,286 @@
+# BGE-base for NFCorpus in Database Vector Stores
+This guide contains instructions for running a BGE-base baseline for NFCorpus in the following databases:
+
+- [DuckDB](#duckdb)
+- [ChromaDB](#chromadb)
+- [Weaviate](#weaviate)
+
+## Indexing
+Assuming you have completed [this guide](https://github.com/castorini/pyserini/blob/master/docs/experiments-nfcorpus.md) and fetched the data, we start by encoding the corpus and queries to obtain embeddings. We will feed these embeddings into the vector stores directly. 
+
+```bash
+mkdir indexes/nfcorpus.bge-base-en-v1.5
+
+python -m pyserini.encode \
+  input   --corpus collections/nfcorpus/corpus.jsonl \
+          --fields title text \
+  output  --embeddings indexes/nfcorpus.bge-base-en-v1.5 \
+  encoder --encoder BAAI/bge-base-en-v1.5 --l2-norm \
+          --device cpu \
+          --pooling mean \
+          --fields title text \
+          --batch 32
+
+mv indexes/nfcorpus.bge-base-en-v1.5/embeddings.jsonl indexes/nfcorpus.bge-base-en-v1.5/corpus_embeddings.jsonl
+
+python -m pyserini.encode \
+  input   --corpus collections/nfcorpus/queries.jsonl \
+          --fields text \
+  output  --embeddings indexes/nfcorpus.bge-base-en-v1.5 \
+  encoder --encoder BAAI/bge-base-en-v1.5 --l2-norm \
+          --device cpu \
+          --pooling mean \
+          --fields text \
+          --batch 32
+
+mv indexes/nfcorpus.bge-base-en-v1.5/embeddings.jsonl indexes/nfcorpus.bge-base-en-v1.5/query_embeddings.jsonl
+```
+
+## DuckDB
+Let's start with DuckDB. Install it with:
+```bash
```

Comment:
as a convention, add a blank line between sentence and code? all throughout please.